src/kaltura-player.js

```diff
@@ -452,6 +453,21 @@ class KalturaPlayer extends FakeEventTarget {
     this._localPlayer.setLogLevel(level, name);
   }
 
+  getThumbnail(time?: number): ?ThumbnailInfo {
+    if (this._thumbnailManager) {
+      if (!time) {
+        // If time isn't supplied, return thumbnail for player's current time
+        if (!isNaN(this.currentTime)) {
+          time = this.currentTime;
+        } else {
+          return null;
+        }
+      }
+      time = this.isLive() ? time + this.getStartTimeOfDvrWindow() : time;
```

Comment:
@OrenMe this is an example where we need the live absolute time (the video element time) and to separate the APIs.
For now I did the transformation here, but I'm still debating if it should be on the UI/application side.
Please give your thoughts.